lib/cartodb/url_utils.rb

```
@@ -0,0 +1,32 @@
+require 'uri'
+require 'cgi'
+
+module CartoDB
+  module UrlUtils
+
+    # Return GET parameters for a url as a Hash
+    # of parameter names as strings to Arrays of
+    # values.  Note that the return value for a
+    # nonexistent parameter name is an empty Array
+    # as opposed to `Nil`.
+    # @param url String
+    # @return Hash
+    def get_url_params(url)
+      CGI.parse(URI.parse(url).query)
+    end
+    module_function :get_url_params
+
+    # Return values of a GET paramter identified
+    # by the specified param_name, or an empty
+    # array if no values for the param_name are
+    # specified.
+    # @param url String
+    # @param param_name String
+    # @return Array of String
+    def get_url_param(url, param_name)
+      get_url_params(url)[param_name]
+    end
+    module_function :get_url_param
+
+  end
+end
```

Missing EOF line

Good job on extracting this. There's a nice opportunity to test it:

```ruby
# in spec/lib/cartodb/url_utils_spec.rb

require_relative '../../spec_helper'

describe(CartoDB::UrlUtils) do
  describe('#get_url_params') do
    it 'handles malformed urls' do
      expect(CartoDB::UrlUtils.get_url_params('foo').to not_raise
    end

    ...
  end
end
```

---

@gfiorav Agree with the addition of the test, I'll write those as well.  If we go with what @vishalya suggested, maybe we should raise an exception if they pass an invalid url string. I imagine a lot of the methods a url helper would provide should not be expected to handle invalid urls, though `get_url_params` could.